lab-christian/model/user.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
```

Comment:
Another really good thing you can do here is to make a `User.create` method that will go through the whole process for you and make a user, without you having to manually do it in your routes and in your tests.